src/Fixer/ControlStructure/SwitchCaseSemicolonToColonFixer.php

```diff
@@ -110,4 +113,20 @@ protected function fixSwitchCase(Tokens $tokens, $index)
             $tokens[$index] = new Token(':');
         }
     }
+
+    /**
+     * @param int $index
+     */
+    protected function fixSwitchDefault(Tokens $tokens, $index)
+    {
+        do {
+            if ($tokens[$index]->equalsAny([':', ';', [T_DOUBLE_ARROW]])) {
```

Comment:
What puzzles me a bit: I would have expected that I could use `'=>'` instead of `[T_DOUBLE_ARROW]`, but that does not work. 🤔

Reply:
Yeah @derrabus ,it's tricky part of [tokenizer](https://www.php.net/manual/en/book.tokenizer.php) that is part of PHP itself.
sometimes, it returns an string, sometimes array with constant. actually `[T_DOUBLE_ARROW]` is shortcut for `[T_DOUBLE_ARROW, '=>']` (but as `T_DOUBLE_ARROW` can be represented only by `=>`, then the 2nd value can be skipped)

ref https://3v4l.org/2beJ3 , look for index 4 for `;` token